desktop/ui/src/main/java/org/datacleaner/windows/AbstractDatastoreDialog.java

```diff
@@ -49,6 +52,27 @@
     private final JButton _saveButton;
     private final JButton _cancelButton;
     private final UserPreferences _userPreferences;
+    
+    protected final JXTextField _datastoreNameTextField;
```

Comment:
I think this is too broad. Keep in mind that AbstractDatastoreDialog is a super class of ALL datastore dialogs. So if you want to centralize the datastore name widget (which is doable) then it needs adaption in 10-15 other subclasses too :-)